### ISSUE TITLE
Bug: San Mateo missing from May Reports

### ISSUE THREAD

**Issue by hunterowens** [cal-itp/reports#160]
During a meeting w/ SPUR / MTC / Bay Area folks, Transit manager from SamTrans was wondering why San Mateo County Transit is missing from the may reports. Any ideas?

**Comment by lauriemerrell** [comment]
Ah, yup. The reports logic is set up to only include feeds where:

* They have `feed_info.txt`
* They have `calitp_url_number = 0`

For SamTrans, their `url_number 0` feed (`http://www.samtrans.com/Assets/GTFS/samtrans/ST-GTFS.zip?v=1114`) does not contain `feed_into.txt`. So, it is failing the criteria. We also track the MTC 511 SamTrans feed (`http://api.511.org/transit/datafeeds?api_key={{ MTC_511_API_KEY}}&operator_id=SM`) which **does** have `feed_into.txt` but that one has `url_number 1`. 

There has been some discussion with @evansiroky and @e-lo about how to handle this for the MTC 511 feeds -- right now we always list the aggregated feed as URL 1 if another feed is available, meaning that the precursor feed is what would be used for the report, except their precursor feed doesn't have `feed_info.txt`. 

I know that @edasmalchi is aware of the limitation about `feed_info.txt`.

Basically there are a few options here:
* Change what URL number we use for the MTC 511 feeds -- list the aggregated feed as feed 0 so it will be reported on (this also will become obsolete as we retire URL number as part of the GTFS schedule pipeline work that is underway)
* Stop requiring `feed_info.txt` to generate a report (I think that Eric already wants to do this) 

Question is mostly prioritization / urgency -- I think that structural fixes for both of these options are already in the works, if we need a shorter-term fix/workaround, we can probably do that too.

**Comment by lauriemerrell** [comment]
^ I guess we could also make a special exception in the report logic for the MTC 511 feeds without rearranging agencies.yml, that would probably be easier and safer in the short term, if we decide that's necessary.

**Comment by hunterowens** [comment]
no need for changes, I can go ask Samtrans to add a feedinfo txt now.

**Comment by edasmalchi** [comment]
Good to know we don't need an urgent change here/anything too hacky in the reports logic.

I would still like to make `feed_info.txt` optional for now as discussed in #153 , I could format the report itself to remind agencies to add if it's missing...

**Comment by holly-g** [comment]
Resolved by #161
